fabric-biome-api-v1/src/main/java/net/fabricmc/fabric/api/biome/v1/BiomeSelectionContext.java

```diff
@@ -143,4 +144,9 @@ default boolean hasBuiltInStructure(ConfiguredStructureFeature<?, ?> configuredS
 	 * <p>If no dimension options exist for the given dimension key, <code>false</code> is returned.
 	 */
 	boolean canGenerateIn(RegistryKey<DimensionOptions> dimensionKey);
+
+	/**
+	 * {@return true if this biome is in the given {@link TagKey}}.
+	 */
+	boolean hasTag(TagKey<Biome> tag);
```

Comment:
Wouldn't it make more sense to name this `isIn` or `isInTag` to match the tag methods? Biome tags contain biomes, not the other way around.

Reply:
This doesn't imply that biomes contain biome tags, however, just that they have them. A biome (or a block, item etc) can definitely "have a tag" IMO - "my bronze ingot has the `#c:bronze_ingots` tag". (`isIn` is also fine since it matches yarn)